gamification/data/labels.py

```diff
@@ -0,0 +1,23 @@
+POSITIVE_POINT_LABELS = (
+    'difficulty/newcomer',
+    'difficulty/low',
+    'difficulty/medium',
+    'difficulty/high',
+    'importance/low',
+    'importance/medium',
+    'importance/high',
+    'importance/critical',
+    'area/documentation',
+    'area/upstream',
+    'area/genericbears',
+    'area/lintbears',
+    'area/nativebears',
```

Comment:
the lists in this file are not necessary.  They can be derived from the data in `points.py` .

DRY

To reduce change to your algorithms, create them as CONSTANTS in `gamification/labels.py` populated via a algorithm rather than static data